### PR TITLE
custom messages ( scoopControl and wheelData)

### PR DESCRIPTION
**Pull request by bgrayd**
Custom messages for wheelData (such as for the current speed of the wheels) and scoopControl (worded vaguely so it can be used for either speed or angle)

### CODE REVIEW

File: cr17/msg/scoopControl.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+float64 arm
```
Comment (daftpwner) [inline cr17/msg/scoopControl.msg]: This variable needs a more meaningful name to indicate it is an angular measurement.

File: cr17/msg/scoopControl.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+float64 scoop
```
Comment (daftpwner) [inline cr17/msg/scoopControl.msg]: More meaningful name.

File: cr17/msg/scoopControl.msg [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (daftpwner) [inline cr17/msg/scoopControl.msg:4]: There should be a header to hold timestamp and frame id information to show when and where these angles have meaning.

File: cr17/msg/wheelData.msg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+float64 frontLeft
+float64 frontRight
+float64 backLeft
+float64 backRight
+
```
Comment (daftpwner) [inline cr17/msg/wheelData.msg:6]: Needs a header for the timestamp.

File: cr17/msg/wheelData.msg [hunk 5]
```diff
@@ -0,0 +1,5 @@
+float64 frontLeft
+float64 frontRight
+float64 backLeft
+float64 backRight
```
Comment (daftpwner) [inline cr17/msg/wheelData.msg]: More meaningful names.  Is this velocity in encoder pulses or m/s?

**Review by daftpwner — CHANGES_REQUESTED** [review]
Needs names that explicitly define the purpose of the variables.  Header's should also be present as this is time and frame oriented information.

**Comment by bgrayd** [comment]
I believe I have addressed the first comments with commit b4e61db5b8cca7f756dd0cb86cf04e22ad268d64

**Review by JacobAMason — APPROVED** [review]
Billy fixed all this, so I'm going to see if this closes the review.

**Comment by JacobAMason** [comment]
Spoiler alert: it doesn't. @daftpwner will have to close it.

**Review by daftpwner — APPROVED** [review]

**Pull request merged**